about.html

```diff
@@ -0,0 +1,204 @@
+            <a href="#events" class="color-2">Events</a>
+          </li>
+          <li>
+            <i class="fa-solid fa-angle-right color-1" aria-hidden="true"></i>
+          </li>
+          <li>
+            <a href="#" class="color-2">IWC</a>
+          </li>
+        </ul>
+        <div class="about-page-header-content">
+          <p class="font-2 color-1 greeting mobile-only">"Breaking the Bias"</p>
+          <h1 class="color-1 font-1 hero">
+            INTERNATIONAL <br />
+            WOMEN'S DAY
+          </h1>
+          <div class="red-line desktop-only"></div>
+          <p class="font-2 color-2 title">
+            We are happy to announce that 2022 International Women's Day will be
+            hosted by the Phenomenal Women in the City of Accra Ghana.
+            <span class="color-1 font-1">SEE YOU THERE!!!</span>
+          </p>
+          <p class="font-2 color-2 mail-header">
+            Please contact us via email for further inquiries
+          </p>
+          <a class="mail" href="mailto:help@iwc.com">questions@iwc.com</a>
+        </div>
+      </header>
+      <section class="summit-details">
+        <h2 class="font-2 color-2 sub-header">
+          International Women's Conference
+        </h2>
+        <div class="red-line"></div>
+        <p class="font-2 color-2">
+          The Women’s Conference Of South Africa Plays A Critical Role In Convening Women Leaders From All Over the World And From All Sectors Of Society, From Girl Scouts To Financial Management.
+          
+        </p>
+        <p class="font-2 color-2 desktop-only">
+          I Was Honored To Be Part Of The Second Annual Conference,
+          Presenting To An Enthusiastic Crowd Of Women And Men Committed To Making IWC A Shining Example Of Equality And Empowerment
+        </p>
+        <span class="cc-logo">
+          <img src="images/women-conference-Logo.jpg" alt="CC logo" />
+        </span>
+      </section>
+      <section id="events" class="past-events">
+        <h3 class="font-2 color-2 sub-header">
+          Previous International women's Conference
+        </h3>
+        <div class="red-line"></div>
+        <p class="color-2 font-2">
+          Take a look at the last two CC Summits that took place in Nigeria and
+          South Africa
+        </p>
+        <div class="previous-events">
+          <div class="e-2019 event">
+            <h4 class="font-2">2019</h4>
+            <p class="font-2">
+              International Women's Conference in Abuja, Nigeria
+            </p>
+          </div>
+          <div class="e-2021 event">
+            <h4 class="font-2">2021</h4>
+            <p class="font-2">
+              International Women's Conference in Pretoria, South Africa
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="partners mobile-only">
+        <h6 class="font-2 color-3">Partners</h6>
+        <div class="red-line"></div>
+        <div class="partner-logo">
```

Comment:
- [ ] Please increase the gap between partners and make sure it matches the design on the template.

### Your design
![image](https://user-images.githubusercontent.com/83803414/158951707-68cdcd61-38e0-4ca5-863b-10bb8f41858f.png)

### Behance design

![image](https://user-images.githubusercontent.com/83803414/158951685-c0638731-d39f-4e80-8726-42ec6a56f53f.png)